openlibrary/components/IdentifiersInput.vue

```diff
@@ -115,40 +114,37 @@ export default {
     },
 
     computed: {
-        popularEditionConfigs: function() {
-            if (this.edition_popular) {
-                const popularConfigs = JSON.parse(decodeURIComponent(this.edition_popular));
-                return Object.fromEntries(popularConfigs.map(e => [e.name, e]));
-            }
-            return {};
+        parsedConfig: function() {
+            return JSON.parse(decodeURIComponent(this.id_config_string))
         },
-        secondaryEditionConfigs: function() {
-            if (this.secondary_identifiers) {
-                const secondConfigs = JSON.parse(decodeURIComponent(this.secondary_identifiers));
-                return Object.fromEntries(secondConfigs.map(e => [e.name, e]));
+        popularIdDict: function() {
```

Comment:
And to keep the naming consistent, let's renaming the property to `popular_ids_string` ; that way any time we have a JSON/string input, it ends with `foo_string`, and then the parsed non-string version is just `foo`.